src/thunderbots/software/test/test_util/rostest_util.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <chrono>
+#include <optional>
+#include <string>
+#include <thread>
+
+namespace RosTest
```

Comment:
Could we rename this to `RosTestUtil` to:
a) better match the file name
b) be less likely to conflict with namespaces from other projects

Reply:
fixed